elm/config/dask_settings.py

```diff
@@ -11,7 +11,7 @@
 from multiprocessing.pool import ThreadPool
 from multiprocessing import Pool
 from dask.threaded import get as dask_threaded_get
-from dask.async import get_sync
+from dask.local import get_sync
```

Comment:
@gbrener One more small change needed (apologies for forgetting this in my commits).  In `elm.pipeline.pipeline` below the:

```
if 'predict' in sklearn_method:
    X = args[0]
```
snippet, do:
```
kwargs.pop('sample_weight', None)
```
`sample_weight` only is needed for fitting, not predicting.  Can you make an issue for us to better test the usage of `sample_weight`.  This has come up now due to low test coverage on its usage.  We can defer addressing that test coverage until after we address #171